src/System.Reflection.Primitives/src/System.Reflection.Primitives.csproj

```diff
@@ -12,11 +12,35 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcore50_Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcore50_Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcore50aot_Debug|AnyCPU'" />
```

Comment:
What is special about the netcore50 configuration? Can it be merged with the default on and we can change the netcore50aot configuration to be netcore50, as don't see any specific System.Private.CoreLib references.